amq-message-age/README.md.jinja

```diff
@@ -10,17 +11,21 @@ Metadata associated with the ActiveMQ message age listener can be found <a targe
 - [Metrics](#metrics)
 - [License](#license)
 
+## OVERVIEW
+
+SignalFx uses ActiveMQ Message Age to measures the age of messages in ActiveMQ queues and publish the results to SignalFx.
+
 ### DESCRIPTION
 
-This tool measures the age of messages in ActiveMQ queues, and publishes the results to SignalFx. It actively inspects the messages that are waiting to be delivered in each queue, and therefore is especially useful for detecting messages that are "stuck" in ActiveMQ queues and unable to be delivered.
+ActiveMQ Message Age Listener actively inspects the messages that are waiting to be delivered in each queue, and therefore is especially useful for detecting messages that are "stuck" in ActiveMQ queues and unable to be delivered.
```

Comment:
Is the name of the tool ActiveMQ Message Age or ActiveMQ Message Age Listener?

Reply:
ActiveMQ Message Age Listener